roles/openstack-stack/templates/heat_stack.yaml.j2

```diff
@@ -130,6 +215,33 @@ resources:
       router_id: { get_resource: router }
       subnet_id: { get_resource: subnet }
 
+{% if use_kuryr|default(false)|bool %}
+  interface:
+    type: OS::Neutron::RouterInterface
+    properties:
+      router_id: { get_resource: router }
+      subnet_id: { get_resource: pod_subnet }
+
+  service_router_port:
+      type: OS::Neutron::Port
+      properties:
+        network: { get_resource: service_net}
+        fixed_ips:
+          - subnet: { get_resource: service_subnet }
+            ip_address: {{ service_subnet_gateway }}
+        name:
+          str_replace:
+            template: openshift-ansible-cluster_id-service-subnet-router-port
+            params:
+              cluster_id: {{ stack_name }}
+  interface:
```

Comment:
The name here ("interface") conflicts with the "interface" resource above "service_router_port"